src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3668,12 +3670,10 @@ namespace HVACMultiSpeedHeatPump {
             }
         }
 
-        // calculate sensible load met using delta enthalpy at a constant (minimum) humidity ratio)
-        MinHumRat = Node(MSHeatPump(MSHeatPumpNum).NodeNumOfControlledZone).HumRat;
-        if (Node(OutletNode).Temp < Node(MSHeatPump(MSHeatPumpNum).NodeNumOfControlledZone).Temp) MinHumRat = Node(OutletNode).HumRat;
-        LoadMet = AirMassFlow * (PsyHFnTdbW(Node(OutletNode).Temp, MinHumRat) -
-                                 PsyHFnTdbW(Node(MSHeatPump(MSHeatPumpNum).NodeNumOfControlledZone).Temp, MinHumRat)) -
-                  MSHeatPump(MSHeatPumpNum).LoadLoss;
+        // calculate sensible load met 
+        Real64 SensibleOutput(0.0);  // sensible output rate
+        SensibleOutput = AirMassFlow * Psychrometrics::PsyDeltaHSenFnTdb2W2Tdb1W1(Node(OutletNode).Temp, Node(OutletNode).HumRat, Node(MSHeatPump(MSHeatPumpNum).NodeNumOfControlledZone).Temp, Node(MSHeatPump(MSHeatPumpNum).NodeNumOfControlledZone).HumRat);
```

Comment:
I can a separate new function it is efficient to use the Pschomteric function "PsyDeltaHSenFnTdb2W2Tdb1W1" directly?

Reply:
Oh, sorry, I didn't recognize this function was the same one used in `CalcTotalSensibleLatentOutput`. OK, I see. 

Makes me wonder if `CalcTotalSensibleLatentOutput` should be in Psychrometrics instead of general?

Reply:
Originally, I added the function into DXCoils but then moved to General as I expended its use to other modules. I am happy to move it Psychrometrics if there is benefit.